main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1276,14 +1276,21 @@ private void updateWatchlistBox() {
             final Button buttonRemove = ButterKnife.findById(view, R.id.remove_from_watchlist);
             final TextView text = ButterKnife.findById(view, R.id.watchlist_text);
 
+            final int watchListCount = cache.getWatchlistCount();
+            String watchListCountString = "";
+            if (watchListCount != -1)
+            {
+                watchListCountString = " (" + watchListCount + ' ' + getString(R.string.cache_watchlist_in_total) + ')';
+            }
+
             if (cache.isOnWatchlist() || cache.isOwner()) {
                 buttonAdd.setVisibility(View.GONE);
                 buttonRemove.setVisibility(View.VISIBLE);
-                text.setText(R.string.cache_watchlist_on);
+                text.setText(res.getString(R.string.cache_watchlist_on) + watchListCountString);
```

Comment:
Please use a single string resource to do this. Different languages may present things in different orders.